charms/metallb-speaker/tests/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""Init mocking for unit tests."""
+
+import sys
+
+sys.path.append('src')
```

Comment:
I think you could tweak the `PYTHONPATH` line in your `tox.ini` files and drop these files.

Reply:
Changing the PYTHONPATH to {toxinidir}/src did not work, along with other attempts, so I will keep the init files to keep it simple